wandb/sdk/data_types/histogram.py

```diff
@@ -34,10 +34,10 @@ class Histogram(WBValue):
         ```
 
     Args:
-        sequence: (array_like) input data for histogram
-        np_histogram: (numpy histogram) alternative input of a precomputed histogram
-        num_bins: (int) Number of bins for the histogram.  The default number of bins
-            is 64.  The maximum number of bins is 512
+        sequence (array_like): input data for histogram
+        np_histogram (numpy histogram): alternative input of a precomputed histogram
+        num_bins (int): Number of bins for the histogram.  The default number of bins
+            is 64. The maximum number of bins is 512
```

Comment:
let's be consistent and remove the types everywhere in the docs and only keep in the function signature

Reply:
I left the types in for Class arguments because they don't show up. lazydocs shows them within `__init__` but we decided to hide those...

Reply:
See this comment: https://github.com/wandb/wandb/pull/9072#issuecomment-2574130726

Reply:
should we expose the init function in that case what do you think?

Reply:
these are arguments to init but here it looks like class attributes